imageio/plugins/tifffile.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015, imageio contributors
+# imageio is distributed under the terms of the (new) BSD License.
+
+""" Storage of image data in tiff format.
+"""
+
+from __future__ import absolute_import, print_function, division
+
+from .. import formats
+from ..core import Format
+
+import numpy as np
+
+try:
+    import tifffile as _tifffile
+except ImportError:
+    import _tifffile
```

Comment:
Oh, ha, I have the system `tifffile` installed so it never got to that line.  (I use 3.4)